### PR TITLE
pt-BR: Catches up with latest PRs on the original EN text

### PR DESCRIPTION
**Pull request by henrif75**
Part of #317 and #893

### CODE REVIEW

File: po/pt-BR.po [hunk 1]
```diff
@@ -7013,11 +7074,11 @@ msgstr ""
 "     }\n"
 "     fn create_default() {\n"
 "         let tmp = Pessoa {\n"
-"             ..Default::default()\n"
+"             ..Pessoa::default()\n"
```
Comment (mgeisler) [inline po/pt-BR.po:7077]: This should match the struct definition above:

```suggestion
"             ..Person::default()\n"
```

Comment (henrif75) [reply po/pt-BR.po:7077]: Actually I had translated the struct name in a previous PR (Person -> Pessoa)

File: po/pt-BR.po [hunk 2]
```diff
@@ -7013,11 +7074,11 @@ msgstr ""
...
 "         };\n"
 "         let tmp = Pessoa {\n"
 "             nome: \"Sam\".to_string(),\n"
-"             ..Default::default()\n"
+"             ..Pessoa::default()\n"
```
[the remark this answers is not in the document]
Comment (henrif75) [reply po/pt-BR.po:7081]: Same here, I had translated the struct name in a previous PR (Person -> Pessoa)

**Review by rastringer — APPROVED** [review]
Thanks Henri! Merged

**Pull request merged**